tests/cases/opacity_inheritance.slint

```diff
@@ -0,0 +1,33 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+export TestCase := Window {
+    preferred-width: 800px;
+    preferred-height: 600px;
+    background: white;
+
+    Rectangle {
+        opacity: 0.5;
+        width: 200px;
+        height: 100px;
+
+        background: red;
+
+        Text {
+            text: "This rectangle should be rose";
```

Comment:
Maybe this example can be in a tests/manual  example?
otherwise tests/cases/manual  so we know we can run it  (and there are some other tests that needs to be done)

The reason not to put it in test/cases is because this will be compiled for each platform, but adds not so much testing value.

Reply:
ACK. I'll move that into tests/manual. The main benefit would be to see if the build passes if `opacity` is used, but there are other tests already that cover that.